### PR TITLE
Move restart to local

### PR DESCRIPTION
**Pull request by felipemadero**
Closes https://github.com/ava-labs/avalanche-network-runner/issues/192

### CODE REVIEW

File: local/network.go [hunk 1]
```diff
@@ -706,6 +706,49 @@ func (ln *localNetwork) removeNode(ctx context.Context, nodeName string) error {
...
+
+	if dbDir != "" {
+		nodeConfig.Flags[config.DBPathKey] = dbDir
+	}
+
+	if err := ln.removeNode(ctx, nodeName); err != nil {
+		return err
+	}
+
+	if _, err := ln.addNode(nodeConfig); err != nil {
```
Comment (felipemadero) [inline local/network.go:745]: is this preserving ports?

File: local/network.go [hunk 2]
```diff
@@ -714,6 +714,55 @@ func (ln *localNetwork) removeNode(ctx context.Context, nodeName string) error {
...
+	if binaryPath != "" {
+		nodeConfig.BinaryPath = binaryPath
+		nodeConfig.Flags[config.BuildDirKey] = filepath.Dir(binaryPath)
+	}
+
+	if whitelistedSubnets != "" {
+		nodeConfig.Flags[config.WhitelistedSubnetsKey] = whitelistedSubnets
+	}
+
+	if dbDir != "" {
```
Comment (holisticode) [inline local/network.go:745]: Why would restarting a node actually allow to change the dbDir? Isn't it the same node?

File: local/network.go (hunk 1 above)
Comment (felipemadero) [reply local/network.go:759]: it is

File: local/network.go (hunk 2 above)
Comment (felipemadero) [reply local/network.go]: double checked. currently there is no need to accept dbdir, rootdir as param to restart.

**Review by holisticode — APPROVED** [review]

**Pull request merged**